Kickstarter-iOS/Views/Controllers/DashboardViewControllerTests.swift

```diff
@@ -47,7 +47,7 @@ internal final class DashboardViewControllerTests: TestCase {
 
         self.scheduler.run()
 
-        FBSnapshotVerifyView(parent.view, identifier: "lang_\(language)_device_\(device)")
+        FBSnapshotVerifyView(parent.view, identifier: "lang_\(language)_device_\(device)", tolerance: 0.05)
```

Comment:
I wonder if the tolerance is too high here considering DashboardViewController is a view controller with such complex UI. As an example, we set the tolerance on RewardPledgeViewControllerTests to 0.0001. I would be happy to pair on this to see if we can find or not a better solution.